src/promptflow-devkit/promptflow/_sdk/data/executable/start_pfs.vbs

```diff
@@ -0,0 +1,4 @@
+DIM objshell
```

Comment:
do we copied installer scripts here? shall this be a move?

Reply:
yes, copy here to support executable run pfs in detachmode when tracing